bolt-design-system/bolt
# NPM | DEV | 'npm run start' creates huge CPU load

## Jira

http://vjira2:8080/browse/BDS-1723

## Summary

Update node and webpack configuration.

## Details

I've done a few tests and the results are shown below. 
![image](https://user-images.githubusercontent.com/16049049/64429046-73162580-d0b5-11e9-866b-bb8ffadd7f56.png)
After updating webpack config and add cache-loader to build, times and CPU overload decrease. The best results i have with node version 12.

## How to test

- Run this code locally. 
- Clear dependencies by run `yarn run clean`
- Optional step: change node version to 12 by:
      - `nvm install 12.9.1`
      - `nvm alias default 12.9.1`
- Install dependencies by run `yarn run setup`
- Run `yarn run start` and check Activity Monitor for overload percentage and listen is you hear fans in your laptop less the usual.
